packages/flutter_tools/test/integration.shard/flutter_immediately_exit_test.dart

```diff
@@ -10,34 +10,36 @@ import 'test_driver.dart';
 import 'test_utils.dart';
 
 void main() {
-  Directory tempDir;
-  final ProjectWithImmediateExit _project = ProjectWithImmediateExit();
-  FlutterRunTestDriver _flutter;
+  group('immediate exit', () {
```

Comment:
Added a group to skip setUp and tearDown as well as the test.  Most of this file is whitespace change.